sqlalchemy_continuum/table_builder.py

```diff
@@ -34,6 +34,10 @@ def reflect_column(self, column):
 
         if not column_copy.primary_key:
             column_copy.nullable = True
+        else:
```

Comment:
I could reverse this, to avoid the negative case being first :P just wanted to keep code changes to a minimum

Reply:
This comment is moot now haha